jsxc.lib.js

```diff
@@ -3045,55 +3035,53 @@ var jsxc;
          var ret = jsxc.gui.template[name];
 
          if (typeof (ret) === 'string') {
-            ret = jsxc.translate(ret);
-
             ret = ret.replace(/\{\{([a-zA-Z0-9_\-]+)\}\}/g, function(s, key) {
                return (typeof ph[key] === 'string') ? ph[key] : s;
             });
 
-            return ret;
+            return $(ret).i18n().text();
```

Comment:
I had to replace this line with <code>return $('<div>').append($(ret).i18n()).html();</code>.